README.md

```diff
@@ -13,15 +12,34 @@ First time repo setup:
 helm repo add linkerd-buoyant https://helm.buoyant.cloud
 ```
 
-Install:
+## Obtain values from the Bcloud UI
+
+To obtain a values file, head over to <https://buoyant.cloud/settings?helm=1>.
+In case you have only one pair of org credentials, you will be prompted with
+a dialog that contains the helm values. Otherwise, you can pick the exact
+credentials pair and click on `Helm usage`. Save the values into `agen-values.yaml`
+
+## Obtain values programatically
+
+If you want to obtain values prograpmatically, you need to know your credentials.
+If you do not know them, you can visit `https://buoyant.cloud/settings` to obtain
+them. When you have the credentials execute the following set of commands:
+
+```bash
+export BUOYANT_CLOUD_CLIENT_ID=<your client id>
+export BUOYANT_CLOUD_CLIENT_SECRET=<your client secret>
+export CREDS=(echo -n $BUOYANT_CLOUD_CLIENT_ID:$BUOYANT_CLOUD_CLIENT_SECRET | base64 --wrap=0)
+export TOKEN=(curl -s -X POST -H "Authorization: Basic $CREDS" "https://api.buoyant.cloud/token" | jq -r .access_token)
+curl -s -H "Authorization: Bearer $TOKEN" "https://api.buoyant.cloud/values.yaml" > agent-values.yaml
+```
+
+## Install:
```

Comment:
remove colon.

i'm tempted to document an additional `Install with params` install flow like:
```bash
export BUOYANT_CLOUD_CLIENT_ID=<your client id>
export BUOYANT_CLOUD_CLIENT_SECRET=<your client secret>
export AGENT_NAME=<your agent name>
helm install --create-namespace --namespace buoyant-cloud --set metadata.agentName=$AGENT_NAME --set api.clientID=$BUOYANT_CLOUD_CLIENT_ID --set api.clientSecret=$BUOYANT_CLOUD_CLIENT_SECRET linkerd-buoyant linkerd-buoyant/linkerd-buoyant
```

this flow arguably eliminates the need to run `Obtain values programmatically` (though I think that's a good flow to document as well).

maybe we should document our recommended flow first, and then document the others as alternatives, something like:

```markdown
# Agent Install
## Obtain values from the Bcloud UI
## Install
## Alternative programmatic install flows
### Obtain values programmatically
### Install with params
# Uninstall
```